ansible/roles/master/templates/kubelet.kubeconfig.j2

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Config
+current-context: kubelet-to-{{ cluster_name }}
+preferences: {}
+clusters:
+- cluster:
+    certificate-authority: {{ kube_cert_dir }}/ca.crt
+    server: https://{{ kube_apiserver_endpoint }}:{{ kube_master_api_port }}
```

Comment:
I think this can change to the following so we do not need to introduce the kube_apiserver_endpoint var:

```
server: https://{% if groups['masters']|length > 1 %}{{ kube_apiserver_vip }}{% else %}{{ groups['masters'][0] }}{% endif %}:{{ kube_master_api_port }}
```